README.md

```diff
@@ -4,29 +4,37 @@ A Foreman plugin to make virt-who configuration easier. This provides a simple U
 
 ## Installation
 
-This plugins requires Katello to be installed in your Foreman instance. If it's there, simply install a package with the plugin (rpm only), run migrations and seed by running,
+### Development setup:
 
-dnf install rubygem-foreman_virt_who_configure
-foreman-rake db:migrate
-foreman-rake db:seed
+* This plugin requires Katello to be installed in your Foreman instance. If it's there, simply install a package with the plugin (rpm only), run migrations and seed by running:
+
+`dnf install rubygem-foreman_virt_who_configure`
```

Comment:
The lines here get squished together:
![image](https://github.com/theforeman/foreman_virt_who_configure/assets/14796566/339860f7-597e-4a50-8eae-9f373409ee57)

Reply:
@ianballou let me know how this looks?